biz.aQute.remote/test/biz/aQute/remote/AgentTest.java

```diff
@@ -23,7 +24,6 @@
 import aQute.remote.api.Event;
 import aQute.remote.api.Supervisor;
 import aQute.remote.util.AgentSupervisor;
-import junit.framework.TestCase;
```

Comment:
This is the cause of the build error.  Also during testing we found that the timeout isn't really working as advertised, likely because `Socket` has some built in timeout.